src/main/java/org/folio/service/impl/TlrSettingsServiceImpl.java

```diff
@@ -39,6 +43,10 @@ public Optional<TlrSettings> updateTlrSettings(TlrSettings tlrSettings) {
       .findFirst()
       .map(entity -> tlrSettingsMapper.mapEntityToDto(
```

Comment:
Shouldn't we create TLR settings if they don't exist?
If update fails in one of the tenants, shouldn't we continue updating settings in the rest of the data tenants?

Reply:
If tlrSettings entity doesn't exist, it will be created. If one of the tenants fails, it will not affect the others

Reply:
In the central tenant, the tlrSettings entity is always present as it's inserted by an SQL script